Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ruby:2.5.1-alpine
+MAINTAINER apply for legal aid team
+
+RUN apk --update add --virtual build-dependencies \
+                                    build-base \
+                                    libxml2-dev \
+                                    libxslt-dev \
+#                                    postgresql-dev \
+                                    sqlite-dev \
+                                    nodejs \
```

Comment:
What necessitates NodeJS being installed?

Reply:
nothing it has now been removed.  I copied it over form another config i had for a web project.